src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -69,12 +69,12 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
             var method = compilerFailFast.GetMethod(nameof(FailFast.OnFatalException), BindingFlags.Static | BindingFlags.NonPublic);
             property.SetValue(null, Delegate.CreateDelegate(property.PropertyType, method));
 
-            _workspace = _componentModel.GetService<VisualStudioWorkspace>();
```

Comment:
This change still does not guarantee that option persisters are created before Workspace gets created, as some other component can MEF import VisualStudioWorkspace. We need to ensure we do not invoke GetOptions from Workspace ctor.